Vungle/VungleRouter.m

```diff
@@ -264,15 +273,24 @@ - (void)requestBannerMrecAdWithPlacementID:(NSString *)placementID delegate:(id<
         [tempDic setObject:delegate forKey:kVungleBannerDelegateKey];
         [tempDic setObject:[NSNumber numberWithInt:BannerRouterDelegateStateRequesting] forKey:kVungleBannerDelegateStateKey];
         [self.bannerDelegates addObject:tempDic];
-
+        
         NSError *error = nil;
-        if ([[VungleSDK sharedSDK] loadPlacementWithID:placementID error:&error]) {
-            NSLog(@"Vungle: Start to load an ad for Placement ID :%@", placementID);
+        if (CGSizeEqualToSize(size, MOPUB_MEDIUM_RECT_SIZE)) {
+            if ([[VungleSDK sharedSDK] loadPlacementWithID:placementID error:&error]) {
+                NSLog(@"Vungle: Start to load an ad for Placement ID :%@", placementID);
```

Comment:
Should this be `NSLog` or `MPLogInfo`? I see that existed previously, but wanted to point this out (also, see line 288). @ashinagawa - thoughts?

Reply:
@crbinsf @ashinagawa Got it, I will replace all NSLogs with MPLogInfos :)